packages/redux/src/products/reducer/__tests__/outfits.test.ts

```diff
@@ -0,0 +1,134 @@
+import { mockProductId } from 'tests/__fixtures__/products';
+import { productsActionTypes, ProductsOutfitsState } from '../..';
+import { toBlackoutError } from '@farfetch/blackout-client';
+import reducer, { getError, getIsLoading, INITIAL_STATE } from '../outfits';
+
+const mockAction = { type: 'foo' };
+const meta = { productId: mockProductId };
+let initialState: ProductsOutfitsState;
+
+describe('outfits redux reducer', () => {
+  beforeEach(() => {
+    initialState = reducer(INITIAL_STATE, mockAction);
+  });
+
+  describe('error() reducer', () => {
+    const error = 'An error occurred';
```

Comment:
Prefer to always use `Error` instances instead of using strings, like this: `new Error('An error occurred)`

Reply:
Fixed. Please review